app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -95,7 +95,7 @@ angular.module('openshiftConsole')
             _.set(pvc, 'spec.selector.matchLabels', selectorLabel);
           }
         }
-        if ($scope.claim.storageClass ) {
+        if ($scope.claim.storageClass && $scope.claim.storageClass.metadata.name !== "No Storage Class") {
```

Comment:
The controller probably shouldn't know this detail of the pvc directive's implementation, but we can fix in a follow on.